front-end/src/dashboard/main.jsx

```diff
@@ -80,6 +81,15 @@ class dashboard extends React.Component {
               </div>
             )
             break
+          case 'blank_panel':
+            columns.push(
+              <div className='col' key={'chart-' + i + '-' + j}>
+                <ErrorBoundary>
+                  <BlankPanel width={config.width} height={config.height} />
```

Comment:
Im curious, do we need a responsive rechart container here? what if its just skip it and push only the parent div elemenr

Reply:
It's not strictly necessary but in my testing I found out that the Recharts responsive container provided a cleaner, better looking UI when using cellphones and tablets.